### PR TITLE
Global state with React blog

### PR DESCRIPTION
**Pull request by davidkpiano**

### CODE REVIEW

File: blog/2024-01-23-xstate-react-global-state/index.mdx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+}
+```
+
+## Global effects
+
+Actors are not limited to being state stores. They can also be used to manage side effects, like HTTP requests, or to trigger side effects from within the actor. Because of this, you may not want actors to start immediately. You can use the `.start()` method to start the actor at an appropriate time, such as when the app mounts.
+
+:::tip
+
+If an actor is already started, calling `.start()` again will not do anything. This makes it safe to call in `useEffect()`.
```
Comment (kevinmaes) [inline blog/2024-01-23-xstate-react-global-state/index.mdx:79]: Such an important point about being "safe" for `useEffect` and a helpful tip. 

Is there anything we can mention if users are wondering if an actor ever needs to be stopped, for example, on component unmount, or if they're worried about memory leaks or anything like that?

File: blog/2024-01-23-xstate-react-global-state/index.mdx [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+```tsx
+// highlight-next-line
+import { countActor } from './countActor';
+
+export function Counter() {
+  // highlight-next-line
+  const count = useSelector(countActor, (state) => state.context.count);
+
+  return (
+    <button onClick={() => countActor.send({ type: 'inc' })}>
```
Comment (mellson) [inline blog/2024-01-23-xstate-react-global-state/index.mdx:119]:
```suggestion
    // highlight-next-line
    <button onClick={() => countActor.send({ type: 'inc' })}>
```

File: blog/2024-01-23-xstate-react-global-state/index.mdx [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+// highlight-next-line
+import { SomeActorContext } from './SomeActorContext';
+
+export function Counter() {
+  // highlight-next-line
+  const someActor = useContext(SomeActorContext);
+  const count = useSelector(someActor, (state) => state.context.count);
+
+  return (
+    <button onClick={() => someActor.send({ type: 'inc' })}>
```
Comment (mellson) [inline blog/2024-01-23-xstate-react-global-state/index.mdx:188]:
```suggestion
    // highlight-next-line
    <button onClick={() => someActor.send({ type: 'inc' })}>
```

**Review by mellson — APPROVED** [review]
Great post 👏🏻 
Nit: Some of the comments inside the JSX are not "proper" JSX comments. It's not a big problem, but Prettier might mess them up if someone else opens and formats the file.

**Comment by kevinmaes** [comment]
This is a really nice post. I think one thing that could be added here is mention of when you need to manually `actor.start()` vs using the React hooks that start the actor for you, and why one would choose one way over the other, since all of these examples are within React (as opposed to just vanilla JS w/o any XState hooks).

**Pull request merged**